tool/teleport/common/teleport.go

```diff
@@ -182,13 +182,23 @@ func onStart(config *service.Config) error {
 func onStatus() error {
 	sid := os.Getenv("SSH_SESSION_ID")
 	proxyHost := os.Getenv("SSH_SESSION_WEBPROXY_ADDR")
-	tuser := os.Getenv("SSH_TELEPORT_USER")
+	teleportUser := os.Getenv("SSH_TELEPORT_USER")
```

Comment:
constant used more than once, makes sense to make it real constant, otherwise LGTM